packages/studiocms/src/schemas/plugins/index.ts

```diff
@@ -223,3 +243,14 @@ export type {
 export function definePlugin(options: StudioCMSPlugin): StudioCMSPlugin {
 	return options;
 }
+
+export type ImageServiceExtraProps = {
+	alt: string;
+	width: number;
+	height: number;
+};
+
+export type StudioCMSImageService = (
+	src: string,
+	props: ImageServiceExtraProps
+) => string | Promise<string>;
```

Comment:
Don't know where this function is but not a huge fan of it returning a promise or not a promise. Tempted to say it should just always be a promise